setup.py

```diff
@@ -48,7 +48,7 @@
         "pydotplus >=2, <3",
         "z3-solver >=4, <5",
         "RecordFlux-language@git+https://github.com/Componolit/RecordFlux-language.git"
-        "@v0.2.0#egg=RecordFlux-language",
+        "@v0.2.1#egg=RecordFlux-language",
```

Comment:
`patchelf` should be removed as dependency in the README.

Reply:
Done in ce76ed1.